subprojects/docs/src/docs/release/notes.md

```diff
@@ -88,6 +88,20 @@ The JAR file is now self-attributing and you don't need to consider whether you
 
 Gradle's [Kotlin DSL](userguide/kotlin_dsl.html) provides an enhanced editing experience in supported IDEs compared to the traditional Groovy DSL — auto-completion, smart content assist, quick access to documentation, navigation to source, and context-aware refactoring.
 
+#### Faster first use
+
+With an empty [Gradle User Home](userguide/directory_layout.html#dir:gradle_user_home), before compiling the first `.gradle.kts` script, Gradle generates the Gradle API Kotlin DSL Extensions JAR.
```

Comment:
Feels a bit "abrupt", like it needs a gentler introduction, more context, not exactly sure what.

Something like "when running a build with an empty Gradle user home (for example first install, ephemeral CI, whatnot)", before we compile ....

Take the next sentence and put it first maybe?

Reply:
Agreed. Does it feels better now with Laura's suggestions applied?

Reply:
Definitely.